### DISCUSSION TITLE
Adding core photos to the existing webmap

### DISCUSSION THREAD

**Discussion by klcummings** [UGS-GIO/rockcore#39] [General]
In our meeting several weeks ago, we discussed adding additional core photos to the current UCRC web app. The spreadsheet that I've compiled for the new iteration of the UCRC database (photos table) has the hyphens and 4-digit alphanumeric codes removed from the well/drill hole APIs. Can I submit this spreadsheet to y'all as-is to add new photos to the existing UCRC webapp? Or will hyphens and 4 digit alphanumeric codes need to be added back to the APIs to use this spreadsheet?

**Comment by nmpayne** [comment]
If the current data has the hyphens and codes I'd rather have them just to keep things consistent.

**Comment by klcummings** [comment]
Here is a spreadsheet that includes photos to be added to the current webapp. I added hyphens and the 4 digit alphanumeric codes to the APIs. I'm not sure what schema was used to add photos to the current webapp - please let me know if this spreadsheet needs to be modified to follow a different schema. 
[20230130_UCRC_photo_list_to_be_added_to_database_20230425 edit_For current UCRC webapp.xlsx](https://github.com/UGS-GIO/rockcore/files/11423622/20230130_UCRC_photo_list_to_be_added_to_database_20230425.edit_For.current.UCRC.webapp.xlsx)

Thanks.
Katie

**Comment by nmpayne** [comment]
Yeah it looks like there is a UWI and API field and one just doesn't have the hyphens. I can easily just populate the API field from UWI. I didn't see a "Type" field in that spreadsheet? 

<img width="634" alt="ucrc" src="https://user-images.githubusercontent.com/15135295/236894615-7a62da68-c0ae-4170-a1a4-2c4cf2d27adb.PNG">

**Reply by klcummings** [reply]
The "type" field is shown as "photo_type" in the provided spreadsheet (all jpg).

**Reply by klcummings** [reply]
I'm not sure what the "WBD" for type means?

**Reply by nmpayne** [reply]
Hmm I'm not seeing that either in the documentation. Maybe sample type? The other value I see is WBW. But in the Field Maps application the photo_type field is populated with those values.

**Comment by klcummings** [comment]
Can we add jpg in the "type" field for the photos included in the list to be added to the current webapp? Or leave that field blank for the photos in the provided spreadsheet?

**Reply by nmpayne** [reply]
Yeah we can add jpg.
